waku/node/v2/waku_types.nim

```diff
@@ -95,7 +81,13 @@ type
     subscribers*: seq[Subscriber]
     pushHandler*: MessagePushHandler
 
-  FilterHandler* = proc(msg: MessagePush) {.gcsafe, closure.}
+  ContentFilter* = object
+    topics*: seq[string]
+
+  ContentFilterHandler* = proc(msg: MessagePush) {.gcsafe, closure.}
+
+  # @TODO MAYBE MORE INFO?
+  Filters* = Table[string, ContentFilterHandler]
```

Comment:
What is string here? Should type alias to a more descriptive type or at least leave a comment

Reply:
I don't quite follow why we can reference ContentFilterHandler directly here whereas previously the value in this table was Filter, which also included the contentFilter

Reply:
We don't need the contentFilter here really, because we do not need to do filtering. What happens here is we store the handler based on the `request_id` string which we get from the filter protocol. All MessagePushes then have that `request_id` so we already know which message goes to which handler.

Reply:
gotcha